tests/cli.rs

```diff
@@ -1,20 +1,58 @@
 #[cfg(test)]
 mod cli {
+    use anyhow::Result;
     use assert_cmd::Command;
+    use http::StatusCode;
     use predicates::str::contains;
-    use std::path::Path;
+    use std::fs::File;
+    use std::io::Write;
+    use std::path::{Path, PathBuf};
+    use wiremock::matchers::path;
+    use wiremock::{Mock, MockServer, ResponseTemplate};
+
+    fn main_command() -> Command {
+        // this gets the "main" binary name (e.g. `lychee`)
+        Command::cargo_bin(env!("CARGO_PKG_NAME")).expect("Couldn't get cargo package name")
+    }
+
+    fn fixtures_path() -> PathBuf {
+        Path::new(module_path!()).parent().unwrap().join("fixtures")
+    }
+
+    // TODO: duplicate of test_utils
```

Comment:
Can we remove those mock server functions and import them from `test_utils` instead or is there a blocker somewhere? In that case I'd rephrase the todo a bit and describe the problem to make it easier to fix in the future. 😉

Reply:
Yeah I tried to fix this, but:

https://github.com/lycheeverse/lychee/pull/35#discussion_r533652892

It might require `lib.rs` to fix properly.